.github/actions/slack-message/action.yml

```diff
@@ -40,6 +40,12 @@ inputs:
     description: 'True if the message is URL encoded'
     required: false
     default: 'true'
+  thread_ts:
```

Comment:
What do you think of writing the full word instead of an abbreviation?. `thread_timestamp` (also other occurrences).